frontend/src/metabase/query_builder/components/template_tags/TagEditorParam.jsx

```diff
@@ -173,10 +176,26 @@ export default class TagEditorParam extends Component {
                 isInitiallyOpen={!tag.dimension}
               />
             )}
+            {hasSelectedDimensionField && !hasWidgetOptions && (
+              <p className="pb1">
+                {t`Oh no! There are no filter widgets for this field. Select
+                another field or`}{" "}
+                <Link
+                  to={MetabaseSettings.docsUrl(
+                    "users-guide/13-sql-parameters",
+                    "the-field-filter-variable-type",
+                  )}
+                  target="_blank"
+                  className="link"
+                >
+                  {t`read about this issue in the docs.`}
```

Comment:
Splitting the sentence like this and putting part of it in the link probably makes it hard to create a translation cleanly. Is that an issue? Is there some way around that or should I just change the copy?

Reply:
We don't have a better way to handle this currently. Probably best to do it like this though:

```
{t`Oh no! There are no filter widgets for this field. Select another field or ${
  <Link
    to={MetabaseSettings.docsUrl("users-guide/13-sql-parameters", "the-field-filter-variable-type")}
    target="_blank"
    className="link"
  >
    {t`read about this issue in the docs`}
  </Link>
}.`}
```